utils/docker/Dockerfile.ubuntu:18.04

```diff
@@ -35,8 +35,8 @@ RUN apt-get install -y python3 python-yaml python3-yaml pylint3
 RUN apt-get install -y python-paramiko python3-paramiko
 RUN apt-get install -y valgrind rsync
 RUN ln -s /usr/bin/cc /bin/cc
-RUN yum -y install ShellCheck
-RUN yum --enablerepo=epel-testing -y install python2-pygithub.noarch
+RUN apt-get install -y ShellCheck
```

Comment:
Why do we need ShellCheck on Ubuntu?  We do linting on CentOS (currently) and shouldn't need to lint on every platform.  The results *should* be the same (modulo version of linting tools) on every platform.